darwin-types/game-objects/GameObject.ts

```diff
@@ -6,6 +6,7 @@ export interface GameObject {
   id: ObjectId;
   type: string;
   position: Position;
+  moveBlocking: boolean;
```

Comment:
Is there any purpose permitting `moveBlocking` to be set on every instance?

**If not:**
We now have two implementations (one in test and one in app)
I'd suggest to create for each type **one** builder which gets used all over the application (also in tests :v:)